app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/ide.tsx

```diff
@@ -0,0 +1,74 @@
+/** @file  */
+import * as react from "react";
+
+import * as service from "../service";
+
+const IDE_CDN_URL = "https://ensocdn.s3.us-west-1.amazonaws.com/ide";
+
+interface Props {
+  project: service.Project;
+  backendService: service.Backend;
+}
+
+declare global {
+  const enso: { main: (config: object) => void; };
+}
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const Ide = ({ project, backendService }: Props) => {
+  react.useEffect(() => {
+    void (async () => {
+      const ideVersions = await backendService.listVersions({
+        versionType: service.VersionType.ide,
+        default: true,
+      });
+      /* TODO: make latest nightlies configurable again */
+      /* TODO: reload style.css and index.js.gz when opening a different version */
+      const projectIdeVersion = "2023.1.1-nightly.2023.2.9"; // project.ideVersion?.number.value ?? ideVersions[0]!.number.value;
+      const stylesheetLink = document.createElement("link");
+      stylesheetLink.rel = "stylesheet";
+      stylesheetLink.href = `${IDE_CDN_URL}/${projectIdeVersion}/style.css`;
+      const indexScript = document.createElement("script");
+      indexScript.src = `${IDE_CDN_URL}/${projectIdeVersion}/index.js.gz`;
+      document.head.append(stylesheetLink);
+      document.body.append(indexScript);
+    })();
+  }, []);
+
+  react.useEffect(() => {
+    void (async () => {
+      const user = await backendService.getUser();
+      const ideVersions = await backendService.listVersions({
+        versionType: service.VersionType.ide,
+        default: true,
+      });
+      const backendVersions = await backendService.listVersions({
+        versionType: service.VersionType.backend,
+        default: true,
+      });
+      console.log('no...', project);
+      const projectIdeVersion = "2023.1.1-nightly.2023.2.9"; // project.ideVersion?.number.value ?? ideVersions[0]!.number.value;
+      const projectEngineVersion = "2023.1.1-nightly.2023.2.9"; // project.engineVersion?.number.value ?? backendVersions[0]!.number.value;
+      enso.main({
+        /* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
why the lint needs to be disabled?

Reply:
the config object was using `snake_case` because the development environment was using an outdated engine version, forgot to remove it when switching back to `camelCase`